ci-operator/config/konflux-workspaces/workspaces/konflux-workspaces-workspaces-main.yaml

```diff
@@ -0,0 +1,48 @@
+build_root:
+  project_image:
+    dockerfile_path: ci/Dockerfile
+releases:
+  initial:
+    integration:
+      name: "4.14"
+      namespace: ocp
+  latest:
+    integration:
+      include_built_images: true
+      name: "4.14"
+      namespace: ocp
+resources:
+  '*':
+    requests:
+      cpu: 100m
+      memory: 250Mi
+tests:
+- as: e2e
+  cluster_claim:
+    architecture: amd64
+    cloud: aws
+    labels:
+      region: us-west-2
```

Comment:
Try changing this to us-east-2 that is how I'm seeing it used for the [RHTAP installer job](https://github.com/openshift/release/blob/3f303c3dec138a39d9938a5aa9c3c86c5d4023e8/ci-operator/config/redhat-appstudio/rhtap-installer/redhat-appstudio-rhtap-installer-release.yaml#L54) which is succeeding in claiming a cluster.

Reply:
Fixed in c152115.

Reply:
yeah that's because [we don't have 4.14 cluster pool in us-west-2 region yet](https://github.com/openshift/release/tree/2114421a38f67d38e64e4a8b0ef86e3ee528e51e/clusters/hosted-mgmt/hive/pools/rhtap)

we will probably bump the version soon